src/Connector.SqlServer/Connector/SqlServerConnector.cs

```diff
@@ -187,25 +213,35 @@ async Task RenameTable(string currentName, string updatedName, string context)
                 }
             }
 
-            var currentContainer = new Container(id);
-            var updatedContainer = new Container(newName);
+            var currentContainer = new Container(id, StreamMode);
+            var updatedContainer = new Container(newName, StreamMode);
 
-            var tasks = new List<Task> { RenameTable(currentContainer.PrimaryTable, updatedContainer.PrimaryTable, "Data") };
+            var tasks = new List<Task> { renameTable(currentContainer.PrimaryTable, updatedContainer.PrimaryTable, "Data") };
             foreach (var current in currentContainer.Tables)
             {
                 if (await CheckTableExists(executionContext, providerDefinitionId, current.Value.Name))
-                    tasks.Add(RenameTable(current.Value.Name, updatedContainer.Tables[current.Key].Name, current.Key));
+                    tasks.Add(renameTable(current.Value.Name, updatedContainer.Tables[current.Key].Name, current.Key));
             }
 
             await Task.WhenAll(tasks);
 
         }
 
+        public override async Task StoreData(ExecutionContext executionContext, Guid providerDefinitionId, string containerName, IDictionary<string, object> data)
+        {
+            await StoreData(executionContext, providerDefinitionId, containerName, null, DateTimeOffset.Now, VersionChangeType.NotSet, data);
```

Comment:
No need for await/async just return directly

Reply:
Sure? 
![image](https://user-images.githubusercontent.com/8555829/128343400-2d0e535d-5b9b-4c81-b93e-28618b7cee5a.png)